human_analysis/archive/3_gene_DE_analysis.ipynb

```diff
@@ -0,0 +1,1771 @@
+       "      <td>9.112307e-11</td>\n",
+       "      <td>22.257258</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>STX11</th>\n",
+       "      <td>-23.845923</td>\n",
+       "      <td>16.245391</td>\n",
+       "      <td>-16.457504</td>\n",
+       "      <td>4.734689e-14</td>\n",
+       "      <td>9.357849e-11</td>\n",
+       "      <td>22.121231</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>STX2</th>\n",
+       "      <td>-4.236599</td>\n",
+       "      <td>6.620074</td>\n",
+       "      <td>-16.277501</td>\n",
+       "      <td>5.955825e-14</td>\n",
+       "      <td>1.059422e-10</td>\n",
+       "      <td>21.904717</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                logFC    AveExpr          t       P.Value     adj.P.Val  \\\n",
+       "SNRK.AS1    -4.958371   5.900715 -21.028847  2.615538e-16  3.049023e-12   \n",
+       "ITIH5       -3.703407   3.134390 -20.765642  3.428180e-16  3.049023e-12   \n",
+       "HYOU1       15.428655  22.573867  17.943977  7.697954e-15  4.564373e-11   \n",
+       "GATA6      -10.775702   9.814635 -17.441516  1.401198e-14  5.823299e-11   \n",
+       "KIAA0040   -16.443337  16.758748 -17.144691  2.010212e-14  5.823299e-11   \n",
+       "PXDC1      -20.778738  22.482341 -17.103739  2.113736e-14  5.823299e-11   \n",
+       "ITGA8       -7.371572   6.564904 -17.038028  2.291606e-14  5.823299e-11   \n",
+       "SH3BP5.AS1 -14.343534  17.211195 -16.571631  4.098182e-14  9.112307e-11   \n",
+       "STX11      -23.845923  16.245391 -16.457504  4.734689e-14  9.357849e-11   \n",
+       "STX2        -4.236599   6.620074 -16.277501  5.955825e-14  1.059422e-10   \n",
+       "\n",
+       "                    B  \n",
+       "SNRK.AS1    26.905199  \n",
+       "ITIH5       26.662724  \n",
+       "HYOU1       23.820691  \n",
+       "GATA6       23.263321  \n",
+       "KIAA0040    22.926022  \n",
+       "PXDC1       22.879006  \n",
+       "ITGA8       22.803318  \n",
+       "SH3BP5.AS1  22.257258  \n",
+       "STX11       22.121231  \n",
+       "STX2        21.904717  "
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "template_DE_stats = pd.read_csv(\n",
+    "    template_DE_stats_file,\n",
+    "    header=0,\n",
+    "    sep='\\t',\n",
+    "    index_col=0)\n",
+    "\n",
+    "template_DEGs = template_DE_stats[(template_DE_stats['adj.P.Val']<0.001) & \n",
+    "                                  (template_DE_stats['logFC'].abs()>1)]\n",
+    "print(template_DEGs.shape)\n",
+    "template_DEGs.head(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Error in library(EnhancedVolcano) : \n",
```

Comment:
This is due to the earlier cell that installs this library is commented out since I found I didn't need to generate this volcano plot in subsequent runs.